backend/Makefile

```diff
@@ -1,6 +1,26 @@
+.PHONY: db/migrate
 db/migrate:
 	alembic -x db=${DEPLOYMENT_STAGE} -c=./config/database.ini upgrade head
 
+.PHONY: db/remote_migration_init
```

Comment:
These make targets support running DB migrations in the remote dev environment on create / update.